RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -2020,6 +2020,53 @@ boolean uneffect(effect toRemove)
 	return false;
 }
 
+int restoreMeatReserve()
```

Comment:
Might as well call it `meatReserve` and move it to auto_util.ash, we might have use for this elsewhere. Also one thing that just occurred to me that we might want to reserve meat for earlier on is the bitchin' meatcar purchase when in a knoll sign.

Reply:
I considered making it a generalized reserve function when creating it. but I couldn't think where else we would use it, and  also considered that if we do use it elsewhere it would probably be with different assumptions than this one which is assuming about restore... that said, i do not mind making the change you suggested.

Reply:
changed name and moved location. currently thinking on how to implement the bitchin meatcar reserve

Reply:
added bitchin meatcar